src/bodies.cpp

```diff
@@ -634,34 +633,47 @@ void bodies::Box::computeBoundingBox(bodies::AABB& bbox) const
   bbox.extendWithTransformedBox(getPose(), 2 * Eigen::Vector3d(length2_, width2_, height2_));
 }
 
+// for some reason, the O2 optimization screws up this function which then gives wrong results
```

Comment:
You removed the actual `GCC` directives again without further comments, but left the comment saying it's broken.

Please either (1) find the gcc bug (report) and link it here, (2) remove the comment if it's not a problem, (3) add the compiler version you used and reference a concrete test that fails.

Reply:
I was unable to reproduce the bug in recent tests. So it probably "got resolved". I'll delete the comment.